airflow/www/templates/airflow/ti_confirm.html

```diff
@@ -0,0 +1,39 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+
+{% extends base_template %}
+
+{% block content %}
```

Comment:
What do we need a special confirm template for? (How is this different from our current confirm.html template?)

Reply:
1 major change - provision for the action endpoint i.e. `success` or `failed`
1 minor change - got rid of hideen confirmation input., though no harm in keeping it.

Reply:
sorry, another major change is that the iteration is happening on `request.args.items` instead of `request.form.items`

Reply:
I still don't see why we need this separate template - changing the other one to request.args should be safe, and then we don't have a second one.

(Is confirm even used anywhere else?)

Reply:
Ok, will do it into the same file, but have to use if-else..